### PR TITLE
Open source

### PR DESCRIPTION
**Pull request by f-lombardo**
I added some hints for onboarding of external contributors

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Contribute to JaRIKo
+
+Feedback and pull requests are very welcome!
+
+## Ask for help, feature or report a bug
+
+Feel free to create feature request and bug report in the [issues](https://github.com/smeup/jariko/issues).
```
Comment (ftomassetti) [inline CONTRIBUTING.md]: It could make sense to add a way to send us questions:
* a gitter channel
* a slack channel
* an email address

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+or [good first issue](https://github.com/smeup/jariko/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) 
+are supposed to be easy and isolated enough to be done by anyone having interest in contributing.
+
+## Build from source
+
+See [Development guide](docs/development.md)
+
+## Coding standard
+
+Follow the [Kotlin conventions](https://kotlinlang.org/docs/reference/coding-conventions.html)
```
Comment (ftomassetti) [inline CONTRIBUTING.md]: We can also explain that we have a task that can refactor the code for style issues

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+## Open a pull request
```
Comment (ftomassetti) [inline CONTRIBUTING.md]: On GitHub it is also possible to make instructions appear when the user create an issue and maybe also a PR

File: rpgJavaInterpreter-core/src/test/kotlin/com/smeup/rpgparser/db/orient/DBOrientTest.kt [hunk 4]
```diff
@@ -0,0 +1,31 @@
+package com.smeup.rpgparser.db.orient
```
Comment (ftomassetti) [inline rpgJavaInterpreter-core/src/test/kotlin/com/smeup/rpgparser/db/orient/DBOrientTest.kt]: This is unrelated :)

Comment (f-lombardo) [reply rpgJavaInterpreter-core/src/test/kotlin/com/smeup/rpgparser/db/orient/DBOrientTest.kt]: Well, in some sens it' related to the PR, since this is referred by a "up for grab" and "good first issue" issues, that are specifically created for onboarding new contributors

File: CONTRIBUTING.md (hunk 2 above)
Comment (f-lombardo) [reply CONTRIBUTING.md]: Good idea!

File: rpgJavaInterpreter-core/src/test/kotlin/com/smeup/rpgparser/db/orient/DBOrientTest.kt (hunk 4 above)
Comment (ftomassetti) [reply rpgJavaInterpreter-core/src/test/kotlin/com/smeup/rpgparser/db/orient/DBOrientTest.kt]: oh, I see!

File: CONTRIBUTING.md (hunk 1 above)
Comment (f-lombardo) [reply CONTRIBUTING.md]: Gitter an slack are good if they have some volume of traffic.
For the email address, I have to do some checks :-)

File: CONTRIBUTING.md (hunk 3 above)
Comment (f-lombardo) [reply CONTRIBUTING.md]: I created `docs/PULL_REQUEST_TEMPLATE.md`: let's see how it works :-)
